geonode/layers/tests.py

```diff
@@ -2111,22 +2111,73 @@ def test_user_get_the_manage_permissions_for_the_selected_dataset(self):
             if dataset:
                 dataset.delete()
 
-    def _create_arguments(self, perms_type):
+    def test_anonymous_user_cannot_get_edit_permissions(self):
+        '''
+        Given the Anonymous user, we should get an error trying to set "edit" permissions.
+        '''
+        try:
+            expected_perms = {
+                'delete_resourcebase',
```

Comment:
i guess this can be removed since we don't expect to have any permission

Reply:
done